release/airflow/cloud_test.template.json

```diff
@@ -10,7 +10,7 @@
     },
     {
       "name": "gcr.io/istio-testing/istio-builder:0.5.10",
-      "args": [ "./test_release.sh", "-b", "$_BRANCH", "-d", "$_DOCKER_HUB", "-g", "$_GCS_BUILD_PATH", "-v", "$_VERSION" ]
+      "args": [ "./test_release.sh", "-b", "$_BRANCH", "-d", "$_DOCKER_HUB", "-g", "$_GCS_BUILD_PATH", "-v", "$_VERSION", "-s", "$_COMMIT" ]
```

Comment:
change line 5 to
"args": [ "gsutil", "cp", "gs://$_GCS_RELEASE_TOOLS_PATH/*", "." ]

and then  ./manifest.xml contains the istio sha for this particular build, extract it in test_release.sh and pass it to githubctl